src/main/java/com/aws/greengrass/easysetup/GreengrassSetup.java

```diff
@@ -107,7 +113,9 @@
             + "runs setup steps,\n"
             + "\t\t\t\t\t(optional) provisions resources, and starts the software. If false, the software runs only "
             + "setup\n"
-            + "\t\t\t\t\tsteps and (optional) provisions resources. Defaults to true.\n";
+            + "\t\t\t\t\tsteps and (optional) provisions resources. Defaults to true.\n"
+            + "\n\t--trusted-plugin, -tp\t\t(Optional) Path of a plugin jar file. The plugin will be included as "
+            + "trusted plugin in nucleus. Specify multiple times for including multiple plugins.\n";
```

Comment:
If we do not support multiple plugins, why support multiple arguments and have this help text at all? It can confuse customers

Reply:
Based on our security discussion today, do we want this to be a parameter? Should we just load from the trusted plugin dir always, since we know that it is protected with root perms?

Reply:
Nucleus loads plugins from trusted plugin only. This is just a convenience to copy to the trusted plugin dir. It is not a good customer experience if we expect them to put the plugin into trusted plugins directory after nucleus has started (trusted plugin might be created on nucleus start).

Reply:
If it is a convenience, then it isn't required. It isn't hard for the customer to just do the copy into trusted themselves.

Reply:
Removing the parameter from easy setup does not make it any more secure. Even with parameter, you need dir (root) permissions to copy jar to the trusted plugin dir.